### PR TITLE
PBC-2890 Integrate TaxApp for Vertex 1.0

### PR DESCRIPTION
**Pull request by k4emic**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/PBC-2890

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -193,6 +193,7 @@
         "spryker/stores-api": "^1.0.0",
         "spryker/stores-backend-api": "^1.0.0",
         "spryker/stores-rest-api": "^1.2.0",
+        "spryker/tax-app": "^0.1.0",
```
Comment (jaume-spryker) [inline composer.json:196]: I couldn't find this 2 packages listed on the PBC requirements (do we assume are included in some of the spryker-feature packages? If that is the case, how do we know the package included on the feature maps with the required version?):
- spryker/stock-address: ^1.2.0
- spryker-shop/cart-page: ^3.38.0

File: composer.json [hunk 2]
```diff
@@ -138,7 +138,7 @@
         "spryker/oauth-api": "^1.3.0",
         "spryker/oauth-auth0": "^1.1.0",
         "spryker/oauth-backend-api": "^1.4.0",
-        "spryker/oauth-client": "^1.3.0",
+        "spryker/oauth-client": "^1.4.0",
```
Comment (jaume-spryker) [inline composer.json:141]: I don't see spryker/oauth-client requirement on the PBC list of required modules (is there a reason for the upgrade? or is the list on PBC manifest outdated?).

**Review by jaume-spryker — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (k4emic) [reply composer.json:196]: The two packages are already required by other packages with compatible constraints.
```
mads@xps:~/projects/demoshops/b2c-demo-shop$ composer why spryker/stock-address
spryker-feature/inventory-management 202307.x-dev requires spryker/stock-address (^1.1.0) 
spryker/stock-address-data-import    0.1.1        requires spryker/stock-address (^1.0.0) 
mads@xps:~/projects/demoshops/b2c-demo-shop$ composer why spryker-shop/cart-page
spryker-feature/cart 202307.0 requires spryker-shop/cart-page (^3.36.0) 
```
So the package update is only contained as part of the composer.lock file.

File: composer.json (hunk 2 above)
Comment (k4emic) [reply composer.json:141]: `spryker/tax-app` requires `^1.4.0`.

```
mads@xps:~/projects/demoshops/b2c-demo-shop$ composer why spryker/oauth-client
spryker-shop/b2c-demo-shop dev-master requires spryker/oauth-client (^1.4.0) 
spryker/app-catalog-gui    1.3.0      requires spryker/oauth-client (^1.0.0) 
spryker/tax-app            0.1.0      requires spryker/oauth-client (^1.4.0)
```

**Review by jaume-spryker — APPROVED** [review]
LGTM

**Pull request merged**